ReactWindows/ReactNative/UIManager/BaseViewManager.cs

```diff
@@ -105,6 +107,31 @@ public void SetZIndex(TFrameworkElement view, int zIndex)
         }
 
         /// <summary>
+        /// Sets the manipulation mode for the view.
+        /// </summary>
+        /// <param name="view">The view instance.</param>
+        /// <param name="manipulationModes">The manipulation modes.</param>
+        [ReactProp("manipulationModes")]
+        public void SetManipulationModes(TFrameworkElement view, JArray manipulationModes)
+        {
+            if (manipulationModes == null)
+            {
+                view.ManipulationMode = ManipulationModes.System;
+                return;
+            }
+
+            var manipulationMode = ManipulationModes.System;
+            foreach (var modeString in manipulationModes)
+            {
+                Debug.Assert(modeString.Type == JTokenType.String);
```

Comment:
Nope, in this case the input validation is being done by the PropTypes declaration in React.js, so adding conditionals and exceptions are wasted compute cycles and wasted developer cycles to go for code coverage on those exceptions (if and when we add unit tests for ViewManager).

The Debug.Assert here is purely to document expectations.